cli/src/action/certs.rs

```diff
@@ -424,6 +466,45 @@ fn handle_skip(
             return Err(CliError::ActionError("CA does not exist".into()));
         }
     }
+
+    #[cfg(feature = "https-certs")]
+    if rest_api_cert_path.exists() && rest_api_key_path.exists() {
+        info!(
+            "Server certificate exists, skipping: {}",
```

Comment:
This looks like a mismatch - was "REST API certificate exists" intended?

Reply:
Fixed.